cedar-lean/Cedar/Thm/Slicing.lean

```diff
@@ -166,21 +163,20 @@ theorem scope_bound_is_sound (policy : Policy) :
     generalize h₂ : policy.principalScope.scope = s
     cases s <;> simp <;>
     apply satisfied_implies_principal_scope h₁ <;>
-    unfold Scope.bound <;> simp [h₂]
+    simp [Scope.bound, h₂]
   case right =>
     generalize h₂ : policy.resourceScope.scope = s
     cases s <;> simp <;>
     apply satisfied_implies_resource_scope h₁ <;>
-    unfold Scope.bound <;> simp [h₂]
+    simp [Scope.bound, h₂]
 
 /--
 Scope-based bound analysis is sound.
 -/
 theorem scope_analysis_is_sound :
   IsSoundBoundAnalysis scopeAnalysis
 := by
-  unfold IsSoundBoundAnalysis
-  apply scope_bound_is_sound
+  simp [IsSoundBoundAnalysis, scope_bound_is_sound]
```

Comment:
is the new version easier to read?  It seems to me the old version is more explicit and perhaps easier to understand what it's doing / why it works

Reply:
Agree this case is arguable. My preferred proof method is spamming the `simp` tactic and under that metric it's better, but I'm fine with removing this change.

Reply:
+1 for spamming `simp`

Reply:
I guess let's see what @emina says.

Reply:
I also find the more verbose proof perhaps slightly easier to understand, step through, and fix.  But short proofs look nicer :). How about we adopt the shorter proofs, and if we find they are harder to maintain, we go back to the more verbose ones?